WORKSPACE

```diff
@@ -1260,3 +1260,19 @@ go_repository(
     commit = "11bc5ee7ad5de4bf1380f3103eebdd40db99a666",
     importpath = "github.com/prysmaticlabs/ethereumapis",
 )
+
+# TODO: Update this dependency after https://github.com/cloudflare/roughtime/pull/13 is merged.
```

Comment:
This was merged, can you update the commit to reference the upstream repo please?